app/experimenter/visualization/api/v3/views.py

```diff
@@ -8,8 +9,30 @@
 
 from experimenter.experiments.models import NimbusExperiment
 
-# This is the metric label for rows that contain user counts.
-USER_COUNTS_KEY = "identity"
+
+class SIGNIFICANCE(IntEnum):
+    POSITIVE = 0
+    NEGATIVE = 1
+    NEUTRAL = 2
+
+
+class BRANCH_COMPARISON:
+    ABSOLUTE = "absolute"
+    DIFFERENCE = "difference"
+    UPLIFT = "relative_uplift"
+
+
+class METRIC:
+    RETENTION = "retained"
+    SEARCH = "search_count"
+    USER_COUNT = "identity"
+
+
+class STATISTIC:
+    PERCENT = "percentage"
+    BINOMIAL = "binomial"
+    MEAN = "mean"
+    COUNT = "count"
```

Comment:
I'm not sure how I feel about this to be honest. I tried it out and because a lot of those constants are being used to compare to string values in the JSON data from Jetstream, the enum object (e.g. `STATISTIC.MEAN`) cannot directly be compared to a string of `mean`, you have to do `STATISTIC.MEAN.value`

It just seems more verbose for not much gain  ¯\_(ツ)_/¯

Reply:
Oh interesting then yeah just leave it as it is 👍